src/kata4.humanCatDogYears.js

```diff
@@ -1,5 +1,28 @@
 const humanCatDogYears = (number) => {
+let tmp = number; 
```

Comment:
No need to assign `number` to `temp` as we can use `number` throughout the algorithm.